### DISCUSSION TITLE
Function description/docs auto pop up in Normal mode

### DISCUSSION THREAD

**Discussion by mangkoran** [helix-editor/helix#6169] [Q&A]
Hi all! Currently I am still learning about Helix, coming from Neovim and (Doom) Emacs.

I noticed that Helix has LSP function description/docs pop up (not sure what it is called), but only in Insert mode. How can I have this in Normal mode, so that the function description would automatically popped up if I hover the function, like I have with Doom Emacs? I have asked around in Matrix and clement-42 pointed that Helix has `SPC k` to display documentation for item under cursor, but it seems this took more space compared to the pop up in Insert mode. Thank you!

Helix (Normal mode):
![image](https://user-images.githubusercontent.com/57625126/222688753-585a6d4d-1033-4c76-84ec-58f8899cda86.png)

Helix (Normal mode, `SPC k`):
![image](https://user-images.githubusercontent.com/57625126/222689512-35d03dc6-3aae-4902-95e7-7fe7c71a6a1a.png)

Helix (Insert mode and what I want in Normal mode):
![image](https://user-images.githubusercontent.com/57625126/222689324-71e093c6-0e28-44ac-b4ce-8de2cbf6e3ff.png)

Doom Emacs
![image](https://user-images.githubusercontent.com/57625126/222688697-9fedb010-449b-421e-89c7-e82bd495abe9.png)

Edit: What I would like to have is documentation to auto pop up/show *on hover* in Normal mode. Helix already has it but it need manual invoke (`SPC k`), not auto pop up.

**Comment by matthewp** [comment]
Same! This would be very useful as you spend a lot of time reading code, not just writing it.

**Comment by kirawi** [comment]
The command to bind for that is `signature_help`

**Comment by cezhunter** [comment]
space - k
